arcjet-node/index.ts

```diff
@@ -255,7 +259,35 @@ export default function arcjet<
           ExtraProps<Rules>
         >;
 
-        return aj.protect({}, req);
+        const getRequestBody = async () => {
+          try {
+            // If request.body is present then the body was likely read by a package like express' `body-parser`.
+            // If it's not present then we attempt to read the bytes from the IncomingMessage ourselves.
+            if (request.body) {
+              return JSON.stringify(request.body);
```

Comment:
Because its an `any` depending on which middleware you used to create it. The most common cases would be JSON or formurlencoded bodies which both get converted to objects which we'd need to stringify. An alternative would be to iterate through the values, check if they are strings and if so then redact those.

Reply:
I think we need to use `typeof` to check it then. If `typeof request.body === "string"` we don't need to serialize it. If `typeof` object we try to serialize, if `typeof` number, I suppose we also have to stringify (??)

Using `typeof` in TS is your best friend because it narrows the type on both side of the if-statement